shelley/chain-and-ledger/executable-spec/test/Generator/Core/QuickCheck.hs

```diff
@@ -66,6 +68,16 @@ mkKeyPairs n
 traceKeyPairs :: KeyPairs
 traceKeyPairs = mkKeyPairs <$> [1 .. 150]
 
+numCoreNodes :: Word64
+numCoreNodes = 7
+
+coreNodes :: [(CoreKeyPair, KeyPair)]
+coreNodes = [ ( (toKeyPair . mkGenKey) (x, 0, 0, 0, 0)
+              , (toKeyPair . mkKeyPair) (x, 0, 0, 0, 1))
+            | x <- [1001..1000+numCoreNodes]]
```

Comment:
would you mind adding a comment to this that includes the reasoning for this magic number?